src/openklant/migration/client.py

```diff
@@ -35,10 +39,8 @@ def _request(
         except requests.RequestException:
             logger.exception(f"{method} request failed for {url}")
 
-            try:
+            with contextlib.suppress(requests.RequestException, UnboundLocalError):
```

Comment:
I don't really understand why this was the way it was previously to be honest. I think the logger.debug statement should just be in an `else` statement after the except, because that is only evaluated if no exceptions occurred (https://docs.python.org/3/reference/compound_stmts.html#else-clause)

```
try:
    # make request
except requests.RequestException:
    logger.exception()
else:
    logger.debug()
```